sql/src/test/java/io/crate/expression/scalar/arithmetic/TruncFunctionTest.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Licensed to CRATE Technology GmbH ("Crate") under one or more contributor
+ * license agreements.  See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.  Crate licenses
+ * this file to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial agreement.
+ */
+
+package io.crate.expression.scalar.arithmetic;
+
+import io.crate.expression.scalar.AbstractScalarFunctionsTest;
+import io.crate.expression.symbol.Literal;
+import io.crate.types.DataType;
+import io.crate.types.DataTypes;
+import org.junit.Test;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+import static io.crate.testing.SymbolMatchers.isFunction;
+import static io.crate.testing.SymbolMatchers.isLiteral;
+
+public class TruncFunctionTest extends AbstractScalarFunctionsTest {
+
+    @Test
+    public void testNormalizeOnDouble() throws Exception {
+        assertNormalize("trunc(29.1947)", isLiteral(29L));
+        assertNormalize("trunc(-29.1947)", isLiteral(-29L));
+        assertNormalize("trunc(29.1947, 0)", isLiteral(29.0d));
```

Comment:
don't we want to treat `trunc(number, 0)` the same as `trunc(number)`?  At least pgsql behaves like this..